src/ert/config/queue_config.py

```diff
@@ -202,25 +200,29 @@ def from_dict(cls, config_dict: ConfigDict) -> QueueConfig:
                     f"Valid choices are {sorted(VALID_QUEUE_OPTIONS[queue_system])}."
                 )
 
-            queue_options[queue_system].append(
-                (option_name, values[0] if values else "")
-            )
+            value = values[0] if values else ""
+            if (
+                option_name in queue_options[queue_system]
+                and queue_options[queue_system][option_name] != value
+            ):
+                logging.info(
```

Comment:
Should we also log if there are duplicate options with the same value?

Reply:
Hmm, does not sound critical, it is more relevant for the users? (anyway, not in this pr)